bouncerscript/task.py

```diff
@@ -50,6 +50,10 @@ def get_task_action(task, script_config):
     return action
 
 
+def matches(name, pattern):
+    return re.match(pattern, name)
```

Comment:
This works, but means we only support matching from the beginning of the string. If we want to allow for substring matching, we should use `search`. https://docs.python.org/3/library/re.html#search-vs-match

It looks like all of your regexes start with `^`, so this probably won't be an issue.